build/webpack.base.config.js

```diff
@@ -124,11 +124,11 @@ module.exports = {
                             ...getCssLoaders(config),
                             {
                                 loader: 'sass-loader',
-                                ...(config.sassLoaderData ? {
-                                    options: {
-                                        data: config.sassLoaderData,
-                                    },
-                                } : {}),
+                                options: Object.assign({
+                                    sourceComments: false,
+                                }, (config.sassLoaderData ? {
+                                    data: config.sassLoaderData,
+                                } : {})),
```

Comment:
Any reason to move away from the spread operation here?  I find that easier to read at a glance than the Object.assign: 

```
                                options: {
                                    sourceComments: false,
                                    ...(config.sassLoaderData ? {
                                        data: config.sassLoaderData,
                                    } : {}),
                                }
```